app/main.py

```diff
@@ -1,3 +1,16 @@
 def cache(func):
-    # Write your code here
-    pass
+    info = {}
+
+    def wrapper(*args):
+        nonlocal info
+        if args in info:
+            print('Getting from cache')
+            for name, inf in info.items():
+                if name == args:
+                    return inf
+        else:
+            print('Calculating new result')
+            result = func(*args)
+            info[args] = result
+            return result
```

Comment:
here you also make `return info[args]`. So you have the same return in both conditions. Then - make 1 return for both cases